client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/heuristic/HeuristicsForJumps.java

```diff
@@ -43,24 +43,19 @@ public static Truthness getForSingleValueJump(int value, int opcode) {
 
     public static Truthness getForValueComparison(int firstValue, int secondValue, int opcode) {
 
-        double a = firstValue;
-        double b = secondValue;
+        int a = firstValue;
```

Comment:
the reason here for using `double` variables was to avoid overflows with `a-b` computations.
good to refactor this class if some of the distance/heuristics computation are re-used in different places.
but, in that case, please add some tests with inputs like `(Integer.MAX, Integer.MIN)` to check for overflow handling

Reply:
fixed